frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -67,6 +67,9 @@ const NewMainLayoutContent = styled(Grid)(({ theme }) => ({
     margin: '0 auto',
     paddingLeft: theme.spacing(2),
     paddingRight: theme.spacing(2),
+    [theme.breakpoints.up(1856)]: {
+        width: '100%',
+    },
```

Comment:
Don't we have standardized breakpoint sizes? `L`, `XL`, etc?

Reply:
We do, and we use the defaults for most things: https://mui.com/material-ui/customization/breakpoints/ however there's one specifically for `1856` on this component's styles:

https://github.com/Unleash/unleash/pull/8267/files#diff-3a5e0763fe1effa9cd485763fecf0633ce8f0877c9d1971e6bd5c5abe7426364R73